integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/LoadTableCommand.scala

```diff
@@ -186,6 +186,12 @@ case class LoadTableCommand(
           LOGGER.error(ex, s"Dataload failure for $dbName.$tableName")
           throw new RuntimeException(s"Dataload failure for $dbName.$tableName, ${ex.getMessage}")
         case ex: Exception =>
+          if (ex.isInstanceOf[InterruptedException] &&
+              ex.getMessage.contains("update fail status")) {
```

Comment:
It is not good to rely on message inside the exception, can you create one special exception for this case?

Reply:
ok, i will add a new Exception for this, and throw that exception